### DISCUSSION TITLE
nvdoc/VIP-books/

### DISCUSSION THREAD

**Discussion by giscus** [visualaids/nvdoc#14] [文章留言]
# nvdoc/VIP-books/

國立臺灣圖書館無障礙閱讀資源整合查詢系統 02-29261470   臺北市立圖書館視障電子圖書館 02-25148443   彰化師大圖書與資訊處視覺障礙資料小組有聲書線上收聽系統 04-7261041   清華大學數位有聲書資料庫平台 03-5721595   台灣數位有聲書推展學會 DAISY 平台 02-23894915   華文視障電子圖書網 02-77300606   愛盲基金會有聲點字圖書館 02-77258000   雲端千眼平台 02-27177722 #39, #51

https://visualaids.github.io/nvdoc/VIP-books/

**Comment by surfer0627** [comment]
愛盲基金會有聲點字圖書館，其中有個分類「精裝書」，由愛盲有聲書專業志工團隊錄製。聽這些志工讀書，那是享受。

祝閱讀愉快。

http://lib.blindness.org.tw/bookbrowsing/bookList?searchField=CLN&searchInput=HC
